### DISCUSSION TITLE
🚨 My Friend’s GitHub Account Got Suspended! 🚨

### DISCUSSION THREAD

**Discussion by edwardj57** [community/community#138873] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello everyone,

I wanted to share something that’s been going on with my friend. They recently had their GitHub account suspended, and it’s been a bit of a rollercoaster since then.

For those who don’t know, he is super talented when it comes to coding and open-source projects. They’ve contributed to some awesome projects, including my team project. It’s frustrating because all that hard work is now locked away, and they can’t access their repositories or collaborate with others.

He is currently working with GitHub support to figure out what happened. It seems like it might be a misunderstanding, but until it gets sorted out, they’re in a tough spot. It’s a reminder of how important these platforms are for developers and how much we rely on them to connect and share our work.

I just wanted to share this to highlight my friend's situation and let everyone know how much they’ve contributed to our community. Here’s hoping they get their account back soon so they can continue doing what they love!

his github username: **heavenstar0531**
email: `LiteBrite82 removed personal identifying information`  [CoC](https://github.com/community/community/blob/6c83e35151f0af5614797de5edf90e77aa7e3e23/CODE_OF_CONDUCT.md)

Thanks for reading!

**Comment by LiteBrite82** [comment]
Hi @edwardj57 

Thank you for your patience. The community is unable to assist with nor escalate your friends ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of theirs. We appreciate your understanding!

Thank you!

**Comment by edwardj57** [comment]
Thank you. @LiteBrite82 
ticket number: #3002596

**Comment by ebndev** [comment, marked as answer]
Hi @edwardj57,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
